src/components/AlertToast.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from "react";
+import { AlertContext } from "@context/AlertContext";
+
+const AlertToast = () => {
+  const { popup, hidePopup } = useContext(AlertContext);
+  const { isVisible, textMessage, messageType } = popup;
+
+  if (!popup.isVisible) {
```

Comment:
above you have already destructured the `isVisible` you can do this
```suggestion
  if (!isVisible) {
```